biojava-core/src/main/java/org/biojava/nbio/core/alignment/SimpleAlignedSequence.java

```diff
@@ -270,6 +270,19 @@ public AccessionID getAccession() {
 	}
 
 	@Override
+	public boolean equals(Sequence<C>other){
```

Comment:
actually we should have written this as

equals(Object) 

instead of equals(Sequence<C>)